config/locales/en.yml

```diff
@@ -736,13 +746,13 @@ en:
         alert: This order can be reconciled
         badge: Can Reconcile
       ready_for_statement:
-        alert: This order is ready to be statemented
-        badge: Ready for Statement
+        alert: This order is ready to be !statement_downcase!ed
```

Comment:
I think we might need a `statemented_downcase` key here - invoice + ed won't work